README.aspnetcore.md

```diff
@@ -15,10 +15,14 @@ These images contain the runtime only. Use [`microsoft/aspnetcore-build-nightly`
 - `1.1.2`, `1.1`, `1`
     - [`1.1.2-jessie` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/jessie/runtime/Dockerfile)
     - [`1.1.2-nanoserver` (*Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/dev/1.1/nanoserver/runtime/Dockerfile)
-- `2.0.0`, `2.0`, `2`, `latest`
+- `2.0.0`, `2.0`
```

Comment:
yes - one thing of note is this [announcement ](https://github.com/dotnet/dotnet-docker/issues/275) which changed the readme formet used in microsoft/dotnet

Sample usage where docker github repo path is f:\repos\dotnet-docker-nightly
`docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v f:\repos\dotnet-docker-nightly:/repo -w /repo microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20170817154316 generateTagsReadme https://github.com/dotnet/dotnet-docker-nightly/blob/master`